app/models/course/assessment/answer/scribing.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::Scribing < ActiveRecord::Base
+  acts_as :answer, class_name: Course::Assessment::Answer.name
+  has_many :scribbles, class_name: Course::Assessment::Answer::ScribingScribble.name,
+                       dependent: :destroy, foreign_key: :answer_id, inverse_of: :answer
+
+  accepts_nested_attributes_for :scribbles, allow_destroy: true
+
+  def to_partial_path
+    'course/assessment/answer/scribing/scribing'
+  end
+
+  # Specific implementation of Course::Assessment::Answer#reset_answer
+  def reset_answer
+    self.class.transaction do
+      scribbles.clear
```

Comment:
Actually if it's only 1 change there is no need to wrap in a transaction (since any failures to this change would automatically trigger a rollback), although this looks a lot neater. @allenwq any thoughts?